app/templates/views/user-profile.html

```diff
@@ -45,14 +45,9 @@ <h1 class="font-body-2xl margin-bottom-3">User profile</h1>
       {% endcall %}
 
       {% call row() %}
-        {{ text_field('Preferred Timezone') }}
+        {{ text_field('Preferred timezone') }}
         {{ optional_text_field(current_user.preferred_timezone) }}
```

Comment:
![image](https://github.com/user-attachments/assets/d2d11e3f-a17e-4d3d-bfa8-955c70a50723)
If I am on the PST, shouldn't it be PST and not EST?

Reply:
It should be wherever the browser thinks you are.  America/San_Diego, America/Chicago, etc.